app/assets/javascripts/models/beta/story.js

```diff
@@ -273,7 +273,7 @@ export const deserialize = (data, options) => {
     ...story,
     labels: Label.splitLabels(story.labels),
     estimate: story.estimate || '',
-    documents: story.documents.map(document => document.file),
+    documents: story.documents ? story.documents.map(document => document.file) : [],
```

Comment:
Just another way to use optional chaining

```suggestion
    documents: (story.documents ?? []).map(document => document.file),
```